### DISCUSSION TITLE
How does reduce_low_mid_tx_power work ?

### DISCUSSION THREAD

**Discussion by pawol** [egzumer/uv-k5-firmware-custom#149] [General]
How does reduce_low_mid_tx_power work ?
I mean what power levels are with/wothout option enabled.

**Comment by egzumer** [comment]
See in the code, search by the name of the ENABLE_*  option. The power values are divided by 5 and 8 if I remember correctly. How that corresponds to the wattage I don't know. It's copied from 1o11
